parliamentarian/__init__.py

```diff
@@ -1,23 +1,30 @@
 """
 This library is a linter for AWS IAM policies.
 """
+
 __version__ = "1.0.0"
 
 import fnmatch
 import functools
+import importlib.resources
 import json
-import jsoncfg
 import re
+from contextlib import ExitStack
 
-import pkg_resources
+import jsoncfg
 import yaml
 
+#
+file_manager = ExitStack()
```

Comment:
it is not clear to me what we are doing with this.  aren't cotnext managers usually used in 'with' blocks?

Reply:
This is part of the recommended migration from `pkg_resources` to `importlib`.  This is the suggested pattern for files that need to persist for longer than a `with` block should be set up for.  See [here](https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-filename).

That said, I think I missed the bits needed to close the file.  I'll get that set up shortly.

Reply:
i see, it looks like you might only need the ExitStack when you need the file to stick around for the life of the program and outside the context of the with.  however, we just parse them and store the python objects in memory.  so i think we can simply use the 'with' keyword.  you might be able to avoid the file(s) and use https://importlib-resources.readthedocs.io/en/latest/migration.html#pkg-resources-resource-stream to pass a stream directly to json.load()/yaml.safe_load() too